src/style.css

```diff
@@ -1,3 +1,85 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  color: #000;
+  font-family: Roboto, sans-serif;
+}
+
+img {
```

Comment:
You should use classes for selecting rather than tags (body tag is unique so it could be an exception).